src/behaviour.rs

```diff
@@ -26,6 +26,7 @@ use std::{time::Duration, vec};
 
 pub(crate) const MAX_TRANSMIT_SIZE: usize = 4;
 pub(crate) const MEGABYTES: usize = 1048576;
+pub(crate) const PROTOCOL_NAME: &str = "/pchain_p2p/1.0.0";
```

Comment:
These constants' visibility can change to private by removing pub(crate). That's on me, i forgot to remove it.

Reply:
Both identity config and kademlia config uses protocol name, however, while kademlia accepts multiple names, identity accepts only 1. So in the event we have multiple protocol names, which one do we use for identity?

Reply:
I think in our Config, we can change the protocol name into a single String instead of a Vec<String>, just like what we used to do. Only put it into a vector in Kad config.